cf-deploy-teardown/tasks/k8s-ready-state-check.sh

```diff
@@ -33,8 +34,8 @@ status "docker info should show overlay2"
 
 # kube-dns shows 4/4 ready
 
-kube_dns=$(kubectl get pods --all-namespaces | grep -q "kube-dns-")
-[[ $kube_dns == *"4/4 Running"* ]]
+kube_dns=$(kubectl get pods --all-namespaces | grep "kube-dns-")
+[[ $kube_dns =~ 4/4\ *Running ]]
```

Comment:
It says, "find all pods, in the `kube-system` namespace, with a label `k8s-app` of value `kube-dns`."  Otherwise you'll find other pods in that namespace.